### ISSUE TITLE
A bug occurred when creating the graph.

### ISSUE THREAD

**Issue by SnowI5Me** [x64dbg/x64dbg#1819]
version 
OS : windows 7 32bit 
x32dbg : Nov 4 2017

I opened the file using x32dbg And then [g] pressed in the CPUTAB
After a while.., an Warring Message appeared.

[ Warring Message ]
Could not allcate memory(minifump wil be created).
[DUMP File](https://www.sendspace.com/file/ciswfy)

Process termination was requested in an unusual way. program will now generate an exception.

For reference, it worked well on other debuggers.

Thanks in advance for your help.

**Comment by mrexodia** [comment]
What other debuggers did you try that have a graph view?

Without the executable I cannot really know if this is a bug on my side or just you having 2gb of memory and running out when trying to graph something...

**Comment by SnowI5Me** [comment]
"Immunity" and "Ida" was able to create graph.

Here is the file
[File Download Link](https://www.sendspace.com/file/0v1n1a)

**Comment by mrexodia** [comment]
Thanks,

Which function should I graph?

**Comment by SnowI5Me** [comment]
when trying to graph "entrypoint"

**Comment by mrexodia** [comment]
A yes, I already ran `upx -d run.exe` and graphed the function at 0x401000 and it is lagging really hard but didn't crash (thanks I'll look into that as well). Graphing at the entry point indeed runs out of memory!

**Comment by SnowI5Me** [comment]
without releasing "upx", An error occurs when graphing at the entrypoint 
But the error is probably due to a lack of memory.

Thank you for your reply.

**Comment by mrexodia** [comment]
No, exactly the same happened on my computer, it used far too much memory for the size of the function!

**Comment by mrexodia** [comment]
Required fix left to do to close this issue: somehow limit the graph analysis to the current section *or* automatically detect when a lot of 0000000000000000 bytes are being rendered and fold them.
